src/main/java/uk/ac/ebi/eva/pipeline/configuration/validation/ParametersValidatorUtil.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import com.google.common.base.Strings;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Utility to hold the low level checks on strings, dirs, files... parameters
+ */
+public class ParametersValidatorUtil {
+
+    static void checkNullOrEmptyString(String stringToValidate,
```

Comment:
Given that it fails if the string is null or empty, maybe `checkIsNotNullOrEmpty` would be a better name?

Following this style, for other methods I would suggest:

* `checkIsBoolean`
* `checkIsInteger`
* `checkDirectoryExists`
* `checkIsPositiveInteger`